tincr/cad/util/util.tcl

```diff
@@ -1337,6 +1339,18 @@ proc ::tincr::catch_info { cmd } {
 #  namespace import ::tincr::*
 interp alias {} ::tincr::import_all {} namespace import ::tincr::*
 
+proc ::tincr::print_quiet { message quiet {newline ""} } {
```

Comment:
This is not a good way to handle verbose logging IMO. Tincr already has a global boolean (tincr::debug) that allows the user to turn on debug logging. Perhaps you could use tincr::debug or add another global boolean called tincr::verbose to turn on the verbose logging you added. Then, I would change this method to look something like this:
```
proc ::tincr::print_verbose { message {newline 0} } {
    if {$::tincr::verbose} {
        puts -nonewline $message
        if {$newline} {
            puts {}
        }
    }
}
```